src/prelude/array.ts

```diff
@@ -130,7 +130,7 @@ export function cumulativeSum(xs: number[]): number[] {
 }
 
 // Object.fromEntries()
-export function fromEntries(xs: [string, any][]): { [x: string]: any; } {
+export function fromEntries<T>(xs: [string, T][]): { [x: string]: T; } {
```

Comment:
💯

Reply:
せっかくリファクタリングするなら

```suggestion
export function fromEntries<T extends readonly (readonly [PropertyKey, any])[]>(xs: T):
  T[number] extends infer U
    ? (
        U extends readonly any[]
          ? (x: { [_ in U[0]]: U[1] }) => any
          : never
      ) extends (x: infer V) => any
        ? V
        : never
    : never {
```

とかの方が良さそう